pkg/apis/ingress/v1alpha1/types.go

```diff
@@ -31,6 +31,10 @@ type ClusterIngressSpec struct {
 	// scheduling of ingress implementation pods.
 	NodePlacement *NodePlacement `json:"nodePlacement"`
 
+	// DefaultCertificateSecret is the name of a secret containing the
+	// default certificate configuration (tls.{crt,key}).
+	DefaultCertificateSecret *string `json:"defaultCertificateSecret"`
```

Comment:
Would it be possible to look for the name of a well-known secret in the namespace instead of letting the user specify one? Advantage is we could defer introducing new API for the time being while still letting us support the feature.

Reply:
To try and summarize an offline discussion: we keep the field with a default equal to the the name of the internal service's cert secret. This seems more clear (explicit vs. implicitly named reference) and also makes it easier to share a cert between multiple instances.

Reply:
Would just `CertificateSecret` make more sense? In our new model, it's *always* defined (by the user or by a default of the internal service secret).

Or does the "Default" refer to its use in the context of a `Route` lacking inline certs? If so, does that inspire any other possible names?

Reply:
Yeah, I just the put the `Default` to indicate it was the default router certificate. `CertificateSecret` could possibly also work - could just add that it is the default router cert in the comments.